lib/service/Service.ts

```diff
@@ -265,18 +264,34 @@ class Service extends EventEmitter {
   }
 
   /**
-   * Get a list of standing orders from the order book for a specified trading pair.
+   * Get a map between pair ids and its orders from the order book.
    */
-  public getOrders = (args: { pairId: string, maxResults: number }) => {
-    const { pairId, maxResults } = args;
-    argChecks.HAS_PAIR_ID(args);
+  public getOrders = (args: { pairId: string, maxResults: number, includeOwnOrders: boolean }) => {
+    const { pairId, maxResults, includeOwnOrders } = args;
     argChecks.MAX_RESULTS_NOT_NEGATIVE(args);
 
-    const result = {
-      peerOrders: this.orderBook.getPeerOrders(pairId, maxResults),
-      ownOrders: this.orderBook.getOwnOrders(pairId, maxResults),
+    const result = new Map<string, OrderArrays>();
+    const getOrderTypes = (pairId: string) => {
+      const result = this.orderBook.getPeerOrders(pairId, maxResults);
```

Comment:
Maybe call this `peerOrders` or `orders` instead of `result`, mainly to eliminate any confusion between it and the `result` variable in the scope above it.